frontend/frontend.php

```diff
@@ -116,6 +121,9 @@ public function init() {
 
 		$this->links = new PLL_Frontend_Links( $this );
 
+		$this->default_term = new PLL_Default_Term( $this );
```

Comment:
Not sur it's needed on front.
Could we just use `PLL_Admin_Default_Term()` in REST?

Reply:
That's something I discussed with @Chouby and we decided to split the class for the 2 contexts.